### PR TITLE
Cautionary contact endpoint

### PR DESCRIPTION
**Pull request by VladyslavAtamanyuk**
Getting the endpoint ready for use

### CODE REVIEW

File: HackneyRepairs/Controllers/CautionaryContactController.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            _cautionaryContactService = cautionaryContactFactory.build(uhwRepository, cautionaryContactLoggerAdapter);
+            _cautionaryContactLoggerAdapter = cautionaryContactLoggerAdapter;
+            _exceptionLogger = exceptionLogger;
+        }
+
+        // GET Cautionary Contacts
+        /// <summary>
+        /// Gets the cautionary contact notes for a address
+        /// </summary>
+        /// <param name="firstLineOfAddress">First line of the propterty address that the notes are being requested of</param>
```
Comment (LBHMKeyworth) [inline HackneyRepairs/Controllers/CautionaryContactController.cs]: Sort the comments here so it has a reference to Address Reference

File: HackneyRepairs/Tests/Actions/CautionaryContactActionsTests.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace HackneyRepairs.Tests.Actions
+{
+    public class CautionaryContactActionsTests
+    {
+        #region Cautionary Contact Data by First Line of Address
```
Comment (LBHMKeyworth) [inline HackneyRepairs/Tests/Actions/CautionaryContactActionsTests.cs]: Change the first line of address in the region :)

File: HackneyRepairs/Tests/Actions/CautionaryContactActionsTests.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        public async Task find_properties_by_address_returns_a_list_of_properties()
```
Comment (LBHMKeyworth) [inline HackneyRepairs/Tests/Actions/CautionaryContactActionsTests.cs]: Change test name for accuracy

File: HackneyRepairs/Controllers/CautionaryContactController.cs (hunk 1 above)
Comment (VladyslavAtamanyuk) [reply HackneyRepairs/Controllers/CautionaryContactController.cs]: Done sir! :)

File: HackneyRepairs/Tests/Actions/CautionaryContactActionsTests.cs (hunk 2 above)
Comment (VladyslavAtamanyuk) [reply HackneyRepairs/Tests/Actions/CautionaryContactActionsTests.cs]: Done sir! :)

File: HackneyRepairs/Tests/Actions/CautionaryContactActionsTests.cs (hunk 3 above)
Comment (VladyslavAtamanyuk) [reply HackneyRepairs/Tests/Actions/CautionaryContactActionsTests.cs]: Done sir! :)

**Comment by LBHMKeyworth** [comment]
Merging branch

**Pull request merged**